website/src/documentation/dsls/sql/extensions/create-external-table.md

```diff
@@ -308,6 +308,43 @@ Write Mode supports writing to a topic.
 
 Only simple types are supported.
 
+## MongoDB
+
+### Syntax
+
+```
+CREATE EXTERNAL TABLE [ IF NOT EXISTS ] tableName (tableElement [, tableElement ]*)
+TYPE mongodb
+LOCATION 'mongodb://[HOST]:[PORT]/[DATABASE]/[COLLECTION]'
+```
+*   `LOCATION`: Location of the collection.
+    *   `HOST`: Location of the MongoDB server. Can be localhost or an ip address.
+         When authentication is required username and password can be specified
+         as follows: `username:password@localhost`.
+    *   `PORT`: Port on which MongoDB server is listening.
+    *   `DATABASE`: Database to connect to.
+    *   `COLLECTION`: Collection within the database.
+
+### Read Mode
+
+Read Mode supports reading from a collection.
+
+### Write Mode
+
+Write Mode supports writing to a collection.
+
+### Schema
+
+Only simple types are supported. MongoDB documents are mapped to Beam SQL types via `JsonToRow` transform.
```

Comment:
Maybe consider adding a link to the [`JsonToRow` javadoc](https://beam.apache.org/releases/javadoc/2.16.0/org/apache/beam/sdk/transforms/JsonToRow.html). I'm not sure how to handle the version number in that link though. It looks like [in the past](https://github.com/apache/beam/commit/860f5c2439b0e5f73a7a65b714a766fb504f227f#diff-b97eb3d3c732a00577269bc09cd61297L37) we used `{{site.release_latest}}` to keep it up to date. But that also carries the risk that the transform will change name and the link will break.

@apilloud or @kennknowles do you have an opinion on this?

I suppose ideally our static site generator would generate this link and simultaneously validate that JsonToRow (or at least JsonToRow.html) exists. After a quick search I don't see any way to do this with jekyll but maybe I missed something.

Reply:
Actually it looks like there's quite a lot of precedent within `website/src` of using `{{site.release_latest}}` for this purpose (some other places use `current` instead of a version number which I didn't realize was an option. @11moon11 could you link to JsonToRow using that?